wandb/env.py

```diff
@@ -440,3 +443,24 @@ def disable_git(env: Optional[Env] = None) -> bool:
     if isinstance(val, str):
         val = False if val.lower() == "false" else True
     return val
+
+
+def get_launch_queue_name(env: Optional[Env] = None) -> Optional[str]:
+    if env is None:
+        env = os.environ
+    val = env.get(LAUNCH_QUEUE_NAME, None)
+    return val
+
+
+def get_launch_queue_entity(env: Optional[Env] = None) -> Optional[str]:
```

Comment:
Quite unfortunate, makes sense given the lack of rules around characters in entities. So close to being able to take a uri approach which is much cleaner.